src/main/ml-modules/root/com.marklogic.smart-mastering/survivorship/merging/base.xqy

```diff
@@ -2126,7 +2166,7 @@ declare private function merge-impl:construct-merging-element($options-json as o
     let $config := json:config("custom")
       => map:with("camel-case", fn:true())
       => map:with("whitespace", "ignore")
-      => map:with("attribute-names", ("name", "weight", "strategy", "propertyName", "algorithmRef", "maxValues"))
+      => map:with("attribute-names", ("name", "weight", "strategy", "propertyName", "algorithmRef", "maxValues", "maxSources"))
```

Comment:
Added maxSources property to filter by a number of sources instead of a number of values. (Helps with carrying over whole arrays of values)